src/features/search/api/useSearchResults/useSearchResults.ts

```diff
@@ -49,7 +52,9 @@ const useSearchInfiniteQuery = (searchState: SearchState) => {
 
   const { nbHits } = data?.pages[0] || { nbHits: 0 }
 
-  return { data, hits, nbHits, ...infiniteQuery }
+  const { userData } = data?.pages[0] || { userData: [] }
+
```

Comment:
Désolé je viens de voir ça aussi
```suggestion
  const { nbHits, userData } = data?.pages[0] || { nbHits: 0, userData: [] }
```